bats/tests/helpers/utils.bash

```diff
@@ -58,3 +58,20 @@ try() {
         count=$((count + 1))
     done
 }
+
+update_allowed_patterns() {
+    local enabled=$1
+    local patterns=$2
+    cat <<EOF |
+{
+  "version": 6,
+  "containerEngine": {
+    "allowedImages": {
+      "enabled": $enabled,
+      "patterns": [$patterns]
+    }
+  }
+}
+EOF
+        rdctl api settings -X PUT --input -
```

Comment:
Why would you use `cat` here when you can redirect STDIN directly. And more importantly, why would you insert the HEREDOC **in the middle** of the command?

```suggestion
    rdctl api settings -X PUT --input <<EOF
{
  "version": 6,
  "containerEngine": {
    "allowedImages": {
      "enabled": $enabled,
      "patterns": [$patterns]
    }
  }
}
EOF
```

Reply:
It took a bit of experimentation to get the syntax right, because the above doesn't work.

You need `rdctl ... --input - <<EOF...`

And this won't work with `run rdctl ... <<EOF` because shell-lint wants to see an assertion on the line immediately after a `run` statement (which is why I was initially using a temp file).